src/config.rs

```diff
@@ -25,6 +25,9 @@ use tracing::{debug, error, warn};
 
 use crate::clients::chunker::DEFAULT_MODEL_ID;
 
+// Place holder to add default allowed headers
+const DEFAULT_ALLOWED_HEADERS: &[&str] = &[];
```

Comment:
Can we rename to `DEFAULT_PASSTHROUGH_HEADERS`?

```suggestion
// Place holder to add default passthrough headers
const DEFAULT_PASSTHROUGH_HEADERS: &[&str] = &[];
```

Reply:
@gkumbhat nit to keep consistent naming, but I think this change got missed to align with the rename